components/position.ts

```diff
@@ -126,6 +127,14 @@ export class PositionService {
     return (nativeEl.style as KeyAttribute)[cssProp];
   }
 
+  private get window():Window {
```

Comment:
this one can create issues with universal >.<
use https://github.com/valor-software/ng2-bootstrap/blob/development/components/utils/facade/browser.ts